jsonschema/golangci.next.jsonschema.json

```diff
@@ -397,6 +397,7 @@
             "nakedret",
             "nestif",
             "nilerr",
+	    "nilnesserr",
```

Comment:
```suggestion
            "nilnesserr",
```

Reply:
Sometimes @alexandear you are living in a parallel world: why do you think I didn't check the box about the jsonschema?